organization_token.go

```diff
@@ -41,10 +41,17 @@ type OrganizationToken struct {
 	Description string    `jsonapi:"attr,description"`
 	LastUsedAt  time.Time `jsonapi:"attr,last-used-at,iso8601"`
 	Token       string    `jsonapi:"attr,token"`
+	ExpiredAt   time.Time `jsonapi:"attr,expired-at,iso8601"`
+}
+
+// OrganizationTokenCreateOptions contains the options for creating an organization token.
+type OrganizationTokenCreateOptions struct {
+	// Optional: The token's expiration date
+	ExpiredAt *time.Time `jsonapi:"attr,expired-at,iso8601,omitempty"`
 }
 
 // Create a new organization token, replacing any existing token.
-func (s *organizationTokens) Create(ctx context.Context, organization string) (*OrganizationToken, error) {
+func (s *organizationTokens) Create(ctx context.Context, organization string, options OrganizationTokenCreateOptions) (*OrganizationToken, error) {
```

Comment:
Make sure that the `options` here get passed to the [POST request](https://github.com/hashicorp/go-tfe/pull/672/files#diff-f74b71435c0aabb3498f3d2b393e5e62c756990e1fa72b8b0792b6c5a3b5941dR60) that actually create the token!

Reply:
Great catch, thank you!